generate.pl

```diff
@@ -133,7 +133,7 @@ sub die_with_sample {
     die "Couldn't create a temp git repo for $release->{version}" if $? != 0;
     Devel::PatchPerl->patch_source($release->{version}, $dir);
     $patch = qx{
-      cd $dir && git diff
+      cd $dir && git diff --src-prefix=a/ --dst-prefix=b/
```

Comment:
Would `git -c diff.mnemonicprefix=false` here also work?

Reply:
Yes, did not know about that option, applied and pushed :+1: